copyparty/umbrel-app.yml

```
@@ -0,0 +1,38 @@
+manifestVersion: 1
+id: copyparty
+name: copyparty
+tagline: A versatile all-in-one file server for everything
+category: files
+version: "1.19.17"
```

Matches with the updated version of docker image.
```suggestion
version: "1.19.19"
```